third_party/binutils/test_decenc/test_decenc_x86_64.asm

```diff
@@ -106433,5 +106426,13 @@ GLOBAL_LABEL(FUNCNAME:)
 
 #endif /* DISABLED_UNTIL_BUG_3581_IS_FIXED */
 
+        /* monitorx, AMD only. */
```

Comment:
Ditto here: these are part of x86_64_arch_3.s.

Reply:
have it your way (you apparently didn't read my comments), but the PR will grow from 54 to 1k lines changed

Reply:
I would prefer to not move anything and to keep these lines where they were.  But if something is going to be moved then the group needs to be moved: splitting it up will cause confusion with the original binutils.

Reply:
that's what I did, I went back and kept everything inplace as it originally was, grouped, nothing moved, but the diff for expected results will be quite large. tests pass so I'm hoping for a final review for this PR

Reply:
a side note:
whoever commented out the unsupported instructions forgot to replace them with nops.

the problem is that each relative call is disassembled with absolute taget address, so
e8 00 00 00 00  => call $...absolute_address
depends on number of bytes before it, this is the problem I was talking about

best would be to either force the disassembler to use relative target, like call $+0 (if that's possible at all) or fix the unsupported instructions with nop-padding

otherwise anyone who's adding tests for new instructions will have 1k-line changes in xxx-expect

but this outside the scope of this PR

Reply:
Yes, I understand and commented earlier on having hit it before and tradeoffs of this "golden output" approach.  Putting in nops is a good idea.  At this point we are willing to live w/ large-ish diffs in the expect file; could certainly be improved.